test/format/format.sh

```diff
@@ -209,16 +226,34 @@ resolve()
 		log="$dir.log"
 
 		# Skip failures we've already reported.
-		[[ -f "$dir/reported" ]] && continue
+		[[ -f "$dir/$status" ]] && continue
+
+		# Get the proceess ID, ignore any jobs that aren't yet running.
+		pid=`grep -E 'process.*running' $log | awk '{print $3}'`
+		[[ "$pid" =~ ^[1-9][0-9]*$ ]] || continue
```

Comment:
Does running into here mean the `RUNDIR.*` directory has been created, but the process has not started (or not yet logged the `process.*running` text)? Should the directory be cleaned up in this case?

Reply:
It means the process hasn't yet logged the `process.*running` text (or maybe we found a `RUNDIR` directory that isn't ours!?), presumably we should keep going.